pkgs/development/libraries/jemalloc/default.nix

```diff
@@ -50,7 +50,8 @@ stdenv.mkDerivation rec {
   env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.isDarwin "-Wno-error=array-bounds";
   env.NIX_LDFLAGS = lib.optionalString (stdenv.cc.libcxx != null) "-l${stdenv.cc.libcxx.cxxabi.libName}";
 
-  doCheck = true;
+  # Tries to link test binaries binaries dynamically and fails
+  doCheck = !stdenv.targetPlatform.isStatic;
```

Comment:
Are you sure you don't mean hostPlatform?  Target is usually only relevant for compilers.

Reply:
You're right